modules/planning/conf/planning.conf

```diff
@@ -34,3 +34,4 @@
 # --export_chart=true
 # --use_gear_shift_trajectory=true
 # --enable_parallel_hybrid_a=true
+# --enable_parallel_open_space_smoother=false
```

Comment:
New empty line needed

Reply:
> There are some conflicts between this commit and the head of master. I suggest to resolve the conflicts locally and resubmit the pull request.
I just fixed the conflicts. Sorry for the late response.